com_dpattachments/admin/src/Model/AttachmentModel.php

```diff
@@ -227,4 +228,15 @@ public function delete(&$pks)
 
 		return $success;
 	}
+
+	public function getAuthor(int $userId): array {
+		$userInfo = array();
+		$user = Factory::getContainer()->get(UserFactoryInterface::class)->loadUserById($userId);
```

Comment:
Use here the UserFactoryAwareInterface and UserFactoryAwareTrait, so you don't need to call the Factory class.